core/src/avm2/globals/mx/core/bytearrayasset.rs

```diff
@@ -0,0 +1,47 @@
+//! `mx.core.BitmapDataAsset` builtin/prototype
+
+use crate::avm2::activation::Activation;
+use crate::avm2::class::{Class, ClassAttributes};
+use crate::avm2::method::Method;
+use crate::avm2::names::{Namespace, QName};
+use crate::avm2::object::Object;
+use crate::avm2::value::Value;
+use crate::avm2::Error;
+use gc_arena::{GcCell, MutationContext};
+
+/// Implements `mx.core.BitmapDataAsset`'s instance constructor.
+pub fn instance_init<'gc>(
```

Comment:
(btw, these don't need to be `pub`, though for some reason a lot of these in existing code are)

Reply:
Okay; I'll keep it as `pub` until/unless the rest get fixed. This project may need some overall cleanup.